wpilibc/src/test/native/cpp/GeometryTest.cpp

```diff
@@ -0,0 +1,60 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include <cmath>
+#include <iostream>
+
+#include "frc/geometry/Pose2d.h"
+#include "gtest/gtest.h"
+
+constexpr double kTestEpsilon = 1E-9;
+
+TEST(GeometryTest, TransformTests) {
+  const frc::Pose2d initial{0.0, 0.0, frc::Rotation2d::FromDegrees(45.0)};
+  const frc::Pose2d final{5.0, 5.0, frc::Rotation2d::FromDegrees(45.0)};
+  const auto finalRelativeToInitial = final.RelativeTo(initial);
+
+  EXPECT_NEAR(finalRelativeToInitial.Translation().X(), 5.0 * std::sqrt(2.0),
```

Comment:
Did EXPECT_DOUBLE_EQ() not work?

Reply:
https://imgur.com/a/aNJ6KYW, no doesn't seem to